### DISCUSSION TITLE
How do I lower the size at which files can be detected?

### DISCUSSION THREAD

**Discussion by Tanggue** [aclap-dev/video-downloadhelper#849] [Help and Q&A]
### Your question

There are times when media is not detected,
It seems that the file size is small and therefore not detected.

In previous versions, the capacity for detecting files could be lowered.
Where do you lower the file detection capacity in the new version?

### Addon & CoApp version and details

version: 9.0.2.3
target: mozilla
channel: stable

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

There is kind of a cheesy solution to the problem.  In the new VDH 9 user interface there is a setting (under the More settings option) for enabling the legacy UI.  That switches the user interface to the one we are used to from past releases.  That doesn't revert VDH to an older release.  It just gives you access to the new VDH using the old UI.  The threshold settings are still in there.  I don't know if they are actually honored, but the settings are still there.  In my case, those settings are blank.  Not 0.  Empty.  My hope is that would cause VDH to recognize everything.  I don't know if that's true, but I am not seeing the problem you are describing.  In any case, it should not be necessary to revert to the legacy UI to get settings with the new VDH 9.  All available settings should be accessible in the new UI.  That's why I say my suggestion is kind of cheesy.

Back in the new UI, there is a check mark option for Show low quality media.  I believe that controls whether certain alternative formats of media are detected.  I don't think that makes any difference to what size of media gets detected.  But you could try toggling that setting to see if it makes a difference.

You should also be using the latest beta.  Click the link you can see at the top of every page of this forum, including this one, labelled Wiki.  That takes you to a page that serves as the closest thing we have to a user manual.  Down the right side of the page, it lists chapters.  Click the chapter that mentions beta versions.  You'll find the beta version of VDH there.
